source/Calamari/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -12,46 +12,137 @@ $ErrorActionPreference = 'Stop'
 # Functions
 # -----------------------------------------------------------------
 
-function Write-VersionTable
+function Log-VersionTable
 {
 	Write-Verbose ($PSVersionTable | Out-String)
 }
 
+function Log-EnvironmentInformation
+{
+	if ($OctopusParameters.ContainsKey("Octopus.Action.Script.SuppressEnvironmentLogging")) {
+		if ($OctopusParameters["Octopus.Action.Script.SuppressEnvironmentLogging"] -eq "True") {
+			return;
+		}
+	}
+
+	Write-Host "##octopus[stdout-verbose]"
```

Comment:
To be consistent, should all of these be `Write-Verbose` instead of `Write-Host` wrapped in the service message? I'm sure there's a good reason I'm missing here...

Hmm, I guess if any of those calls write to the host without you realising, they'll all end up at Verbose - so maybe this is a good approach. :)